src/background/Wallet/Wallet.ts

```diff
@@ -1065,7 +1097,11 @@ export class Wallet {
     const txWithFee = await prepareGasAndNetworkFee(prepared, networks, {
       source: mode === 'testnet' ? 'testnet' : 'mainnet',
     });
-    const transaction = await prepareNonce(txWithFee, networks, chain);
+    // TODO: remove `prepareTransactionType` helper after update to ethers v6
+    // ethers v5 throws error inside `getFeeData` for some chains with too big totalDifficulty param
+    // can be reproduced with https://chainlist.org/chain/30732
+    const txWithFeeAndType = await prepareTransactionType(txWithFee);
```

Comment:
Why are we awaiting? It seems to be sync

Reply:
Ah, really! Good catch!